src/technologies/w.json

```diff
@@ -2079,6 +2079,16 @@
     "scriptSrc": "/wp-content/themes/default/",
     "website": "https://wordpress.org/themes/default"
   },
+  "WordPress Full Site Editing": {
+    "cats": [
+      51
+    ],
+    "description": "Beaver Builder is a drag and drop page builder for WordPress.",
```

Comment:
Full Site Editing enables users to design and customize their entire WordPress website with a block-based editor.